tests/test_configuration.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import annotations
+
+import sys
+import tempfile
+from pathlib import Path
+from unittest.mock import patch
+
+import platformdirs
+
+from variantlib.configuration import ConfigEnvironments
+from variantlib.configuration import VariantConfiguration
+from variantlib.configuration import get_configuration_files
+from variantlib.constants import CONFIG_FILENAME
+from variantlib.models.configuration import VariantConfiguration as ConfigurationModel
+from variantlib.models.variant import VariantFeature
+from variantlib.models.variant import VariantProperty
+
+
+def dict_to_toml(d: dict, indent: int = 0) -> str:
```

Comment:
Can't we use `tomli_w` here? I realize the output is relatively simple but I feel like implementing our own TOML serializer is going a bit far, especially that it'd be only a test dependency.

Reply:
I guess for testing it doesn't matter indeed - I didn't want to add dependencies :D

Reply:
Fixed in e36e93161dc007e0b326bb9d5422111d4bf3192f